e2e/shortcut-action/shortcut-action.e2e-spec.ts

```diff
@@ -58,19 +58,39 @@ describe('ShortcutAction', function() {
   });
 
   it(`should move row up using 'mod+shift+up' shortcut`, () => {
-    let inputElem = page.getChildOfElementByCss(page.getElementById('accelerator_experiments.1.experiment'), 'textarea');
-    let inputElemValue = inputElem.getAttribute('value');
-    inputElem.sendKeys(protractor.Key.chord(mod, protractor.Key.SHIFT, protractor.Key.UP));
-    let movedUpInputElem = page.getChildOfElementByCss(page.getElementById('accelerator_experiments.0.experiment'), 'textarea');
-    expect(movedUpInputElem.getAttribute('value')).toEqual(inputElemValue);
+    let currentFirstRow = page.getValuesOfChildrenById('keywords.0');
+    let currentSecondRow = page.getValuesOfChildrenById('keywords.1');
+    let currentElem = page.getChildOfElementByCss(page.getElementById('keywords.1.keyword'), 'textarea');
+    currentElem.sendKeys(protractor.Key.chord(mod, protractor.Key.SHIFT, protractor.Key.UP));
+    let targetFirstRow = page.getValuesOfChildrenById('keywords.0');
+    let targetSecondRow = page.getValuesOfChildrenById('keywords.1');
+    expect(targetFirstRow).toEqual(currentSecondRow);
+    expect(targetSecondRow).toEqual(currentFirstRow);
+    // Test for issue of overriding the new value with the old one when moving up because of triggering the commitValueChange()
+    // on blur with the old values. As a result the switched cells have the same values
+    expect(targetSecondRow).not.toEqual(targetFirstRow);
   });
 
   it(`should move row down using 'mod+shift+down' shortcut`, () => {
-    let inputElem = page.getChildOfElementByCss(page.getElementById('accelerator_experiments.0.experiment'), 'textarea');
-    let inputElemValue = inputElem.getAttribute('value');
-    inputElem.sendKeys(protractor.Key.chord(mod, protractor.Key.SHIFT, protractor.Key.DOWN));
-    let movedDownInputElem = page.getChildOfElementByCss(page.getElementById('accelerator_experiments.1.experiment'), 'textarea');
-    expect(movedDownInputElem.getAttribute('value')).toEqual(inputElemValue);
+   let currentRow = page.getValuesOfChildrenById('keywords.0');
+   let currentElem = page.getChildOfElementByCss(page.getElementById('keywords.0.keyword'), 'textarea');
+   currentElem.sendKeys(protractor.Key.chord(mod, protractor.Key.SHIFT, protractor.Key.DOWN));
+   currentElem = page.getChildOfElementByCss(page.getElementById('keywords.1.keyword'), 'textarea');
+   // Trigger move down shortcut two time in a row to test issue of not updating tabindexes correctly on sequential
+   // trigger of the shortcut. As a result the shortcut was not working properly.
+   currentElem.sendKeys(protractor.Key.chord(mod, protractor.Key.SHIFT, protractor.Key.DOWN));
+   let targetRow = page.getValuesOfChildrenById('keywords.2');
+   expect(targetRow).toEqual(currentRow);
+  });
+
+  it(`should update tabindexes correctly of the switched rows when moving up/down clicking the button`, () => {
```

Comment:
This test is only checking the up button, so description saying up/down is not correct.

Can you add another test for the down button?